src/main/java/net/consensys/orion/storage/OrionSQLKeyValueStore.java

```diff
@@ -1,88 +0,0 @@
-/*
```

Comment:
So this class isn't used anymore? It doesn't impact Orion's capabilities of using a relational db, does it?
(just checking it wasn't a mistake) :)

Reply:
I moved it down to Apache Tuweni. Check out this class: https://github.com/apache/incubator-tuweni/blob/master/kv/src/main/kotlin/org/apache/tuweni/kv/EntityManagerKeyValueStore.kt

Reply:
and this is how we do it in Orion:
```final JpaEntityManagerProvider jpaEntityManagerProvider = new JpaEntityManagerProvider(dbName);
      return ProxyKeyValueStore.open(
          EntityManagerKeyValueStore.open(jpaEntityManagerProvider::createEntityManager, Store.class, Store::getKey),
          Base64::decode,
          Base64::encode,
          store -> Bytes.wrap(store.getValue()),
          (Bytes key, Bytes value) -> {
            Store store = new Store();
            store.setKey(Base64.encode(key));
            store.setValue(value.toArrayUnsafe());
            return store;
          });```

Reply:
ok, marking this resolved now.